core/controllers/story_editor.py

```diff
@@ -59,60 +59,60 @@ def _require_valid_version(self, version_from_payload, story_version):
                 % (story_version, version_from_payload))
 
     @acl_decorators.can_edit_story
-    def get(self, story_id):
+    def get(self, storyId):
         """Populates the data on the individual story page."""
-        story = story_fetchers.get_story_by_id(story_id, strict=False)
-        topic_id = story.corresponding_topic_id
-        topic = topic_fetchers.get_topic_by_id(topic_id, strict=False)
-        skill_ids = topic.get_all_skill_ids()
-        skill_summaries = skill_services.get_multi_skill_summaries(skill_ids)
-        skill_summary_dicts = [summary.to_dict() for summary in skill_summaries]
+        story = story_fetchers.get_story_by_id(storyId, strict=False)
+        topicId = story.correspondingTopicId
+        topic = topic_fetchers.get_topic_by_id(topicId, strict=False)
+        skillIds = topic.get_all_skill_ids()
+        skillSummaries = skill_services.get_multi_skill_summaries(skillIds)
+        skillSummaryDicts = [summary.to_dict() for summary in skillSummaries]
 
         for story_reference in topic.canonical_story_references:
-            if story_reference.story_id == story_id:
-                story_is_published = story_reference.story_is_published
+            if story_reference.storyId == storyId:
+                storyIsPublished = story_reference.storyIsPublished
 
         self.values.update({
             'story': story.to_dict(),
-            'topic_name': topic.name,
-            'story_is_published': story_is_published,
```

Comment:
That being said, the RHS of this should still be in snake_case, as that represents a variable local to the controller, while LHS should be camelCase as that represents the variable returned to the frontend.